src/libraries/System.IO.Ports/tests/SerialPort/ReadTo.cs

```diff
@@ -691,7 +694,7 @@ private void PerformReadOnCom1FromCom2(SerialPort com1, SerialPort com2, string
             int totalCharsRead;
             int lastIndexOfNewLine = -newLineStringLength;
             string expectedString;
-            bool isUTF7Encoding = com1.Encoding.EncodingName == Encoding.UTF7.EncodingName;
+            bool isUTF7Encoding = com1.Encoding.WebName == UTF7_WEBNAME;
```

Comment:
>com1.Encoding.WebName == UTF7_WEBNAME; [](start = 34, length = 38)

I would prefer comparing the CodePage instead of comparing names.

Reply:
Sure. I'll flow this change throughout the project, and it should also address other reviewers' concerns about adding new consts to these files.